src/Clock.tsx

```diff
@@ -0,0 +1,60 @@
+import { Component } from 'react';
+
+type State = {
+  today: string;
+};
+
+type Props = {
+  clockName: string,
+};
+
+const formateDate = (date: Date): string => {
+  return date.toUTCString().slice(-12, -4);
+};
+
+export class Clock extends Component<Props, State> {
+  state = {
+    today: formateDate(new Date()),
+  };
+
+  timerId = 0;
+
+  componentDidMount(): void {
+    this.timerId = window.setInterval(() => {
+      this.setState({ today: formateDate(new Date()) });
+
+      // eslint-disable-next-line no-console
+      console.info(this.state.today);
```

Comment:
You can write this `  const { today} = this.state;` and then use just `today` in console